packages/components/src/components/chat/Chat.vue

```diff
@@ -235,6 +250,26 @@ export default {
 
       message.sentiment = sentiment;
     },
+    isAutoScrolled() {
+      const { messages } = this.$refs;
+      return messages.scrollTop === this.autoScrolTop;
```

Comment:
This will always evaluate to false due to `autoScrolTop` (sp) being undefined. In fact, it looks like this method is unused.